sdk/tests/invariant.test.ts

```diff
@@ -248,4 +269,240 @@ describe('invariant', async () => {
     const pools = await invariant.getPools(account)
     assert.deepEqual(pools.length, 1)
   })
+  describe('positions', async () => {
+    const lowerTickIndex = -20n
+    const upperTickIndex = 10n
```

Comment:
I like the approach of defining all common configuration constants here. You can also consider moving constants like this:
```ts
const feeTier = newFeeTier({ v: 6000000000n }, 10)
```